spec-0005/index.md

```diff
@@ -25,6 +26,34 @@ intended use-cases, and pseudo-code illustrating its use.
 Discuss how this would be implemented.
 -->
 
+### Build nightly wheels
+
+By building and uploading wheel for your project you make it easier for projects
+that depend on you to test against your latest changes. They can then report problems
+they find.
+
+There are a few steps to implementing this for your project:
+1. Get access to the area that nightly wheels are uploaded to
+2. Setup a CI step that builds wheels for your project
+3. Setup a CI step that uploads wheels to https://anaconda.org/scientific-python-nightly-wheels/
```

Comment:
I would reorder these and put no2 in the first position as no1 and no3 kind of belongs together